### ISSUE TITLE
markupToList

### ISSUE THREAD

**Issue by clojj** [jtdaugherty/brick#209]
If a Markup contains the pair `('\n', ...)`, I think it is not converted correctly by `Brick.Markup.markup` as 
`txt "\n"`

`ch == '\n' = toLines (ls ++ [cur]) [] rest` concatenates `[]` instead of `[('\n', ...)]`

```
markupToList :: (Eq a) => Markup a -> [[(T.Text, a)]]
markupToList (Markup thePairs) = toList <$> toLines [] [] thePairs
    where
        toLines ls cur [] = ls ++ [cur]
        toLines ls cur ((ch, val):rest)
            | ch == '\n' = toLines (ls ++ [cur]) [] rest
            | otherwise = toLines ls (cur ++ [(ch, val)]) rest

        toList [] = []
        toList ((ch, val):rest) = (T.pack $ ch : (fst <$> matching), val) : toList remaining
            where
                (matching, remaining) = break (\(_, v) -> v /= val) rest

```

**Comment by jtdaugherty** [comment]
Can you give an example of the specific input you want to use, the expected outcome, and the actual outcome?

**Comment by clojj** [comment]
markup containing just '\n' are not rendered
I would expect them to render as newlines

![markup-newlines](https://user-images.githubusercontent.com/4367963/49691060-cc5b2380-fb3a-11e8-8cbb-cbe2255e867f.png)

**Comment by jtdaugherty** [comment]
Thanks, that helps! I was able to reproduce the problem.

**Comment by jtdaugherty** [comment]
The issue is now fixed; you can give the fix a try on `master`.

**Comment by jtdaugherty** [comment]
Thanks for the report!

**Comment by jtdaugherty** [comment]
This fix is now released in `0.44.1`.

**Comment by clojj** [comment]
now works as expected
